api/scpca_portal/test/management/commands/test_load_metadata.py

```diff
@@ -0,0 +1,297 @@
+    def test_data_clean_up(self, mock_clean_up_input_data):
+        project_id = "SCPCP999990"
+        self.load_metadata(
+            clean_up_input_data=True,
+            reload_all=False,
+            reload_existing=False,
+            scpca_project_id=project_id,
+            update_s3=False,
+        )
+
+        mock_clean_up_input_data.assert_called_once()
+
+    def test_load_metadata(self):
+        project_id = "SCPCP999990"
+
+        def assert_object_count(*, objects_loaded=True):
+            self.assertEqual(Project.objects.count(), 1 if objects_loaded else 0)
+            self.assertEqual(ProjectSummary.objects.count(), 4 if objects_loaded else 0)
+            self.assertEqual(Sample.objects.count(), 4 if objects_loaded else 0)
+
+        # First, just test that loading data works.
+        self.load_metadata(
+            clean_up_input_data=False,
+            reload_all=False,
+            reload_existing=False,
+            scpca_project_id=project_id,
+            update_s3=False,
+        )
+        assert_object_count()
+
+        project = Project.objects.get(scpca_id=project_id)
+        self.assertProjectData(project)
+
+        # Make sure that reload_existing=False won't add anything new when there's nothing new.
+        self.load_metadata(
+            reload_all=False,
+            reload_existing=False,
+            scpca_project_id=project_id,
+            update_s3=False,
+        )
+        assert_object_count()
+
+        new_project = Project.objects.get(scpca_id=project_id)
+        self.assertEqual(project, new_project)
+        self.assertEqual(project.summaries.first(), new_project.summaries.first())
+        self.assertEqual(project.samples.first(), new_project.samples.first())
+
+        # Make sure purging works as expected.
+        Project.objects.get(scpca_id=project_id).purge()
+
+        assert_object_count(objects_loaded=False)
+
+        # Make sure reloading works smoothly.
+        self.load_metadata(
+            clean_up_input_data=False,
+            reload_all=False,
+            reload_existing=True,
+            scpca_project_id=project_id,
+            update_s3=False,
+        )
+        assert_object_count()
+
+    def test_single_cell_metadata(self):
+        project_id = "SCPCP999990"
+        self.load_metadata(
+            clean_up_input_data=False,
+            reload_all=False,
+            reload_existing=False,
+            scpca_project_id=project_id,
+            update_s3=False,
+        )
+
+        project = Project.objects.get(scpca_id=project_id)
+        self.assertProjectData(project)
+        self.assertFalse(project.has_cite_seq_data)
+        self.assertTrue(project.includes_anndata)
+        self.assertTrue(project.modalities)
+        self.assertEqual(project.multiplexed_sample_count, 0)
+        self.assertEqual(project.organisms, ["Homo sapiens"])
+        # This project contains 3 samples
+        single_cell = 2
+        spatial = 1
+        bulk = 1
```

Comment:
What's the deal with this `bulk` sample? It has scpca_id `SCPCS999994`, it's included in the `samples_metadata.csv` file but has to corresponding `json` file.